comp/snmpscan/def/component.go

```diff
@@ -0,0 +1,29 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2024-present Datadog, Inc.
+
+// Package snmpscan ...
```

Comment:
Could you provide a complete package comment? 😄

Reply:
Oops yeah missed that with the generated tool!